DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1

```diff
@@ -33,13 +33,18 @@ Composite DSC Resource allowing you to specify multiple choco packages in a sing
         [ValidateNotNullOrEmpty()]
         [System.String]
         $Source
+        [parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]

```

Comment:
A boolean can only have $true or $false. If you pass anything empty it won't convert and will throw an (conversion) exception. So this is effectively redundant.